Make ActionCable logging less verbose in development

jankeesvw opened the pull request:
When running the ActionCable server in development I get a lot of log messages, even when I set the application log level to `config.log_level = :info`. ~~With this commit I move the level for ActionCable to `debug` which is more appropriate in my opinion.~~ With this commit I truncate the log message as proposed by @dhh in https://github.com/rails/rails/pull/23709#issuecomment-185260388.

Example of my current output to illustrate the problem: 

![image](https://cloud.githubusercontent.com/assets/167882/13077891/f83dba42-d4bb-11e5-939f-384bfffd10bd.png)

jankeesvw committed:
Truncate ActionCable broadcast message to 300 chars

When running the ActionCable server in development I get a lot of output
in my logs, this commit sets a maximum length of 300 characters for a
broadcast log message.

rafaelfranca commented:
I think this one make sense to be `info` since you want to know when the broadcast is being done. Maybe we should only include the message in the debug mode?

r? @dhh 

cc @matthewd

jankeesvw commented:
I would also be happy to remove the `#{message}` part. If we remove that it would be a lot more readable for me.

Like this:

```ruby
server.logger.info "[ActionCable] Broadcasting to #{broadcasting}"

dhh commented:
I'd stick with info and just truncate the message part so it doesn't go crazy on a big HTML send. Maybe just do the first 300 chars or so.

jankeesvw commented:
Hi @dhh, I changed the commit as you proposed, I also added three dots (`...`) so we have a clue this could be shortened. My log now looks like this (which is much better): 

```
PrinterListChannel transmitting "<div class=\"mod-printers-list\" data-id=\"printer-list\">\n  <table class=\"list-view\">\n    <thead>\n      <tr>\n        <th class=\"lead\">\n          Printers\n        </th>\n        <th>\n          Job\n        </th>\n        <th>\n          Time remaining\n        </th>\n        <th colspan=\... (via streamed from PrinterListChannel)
JobListChannel transmitting "<table class=\"list-view mod-jobs-list\" data-id=\"job-list\">\n  <thead>\n    <tr>\n      <th class=\"lead\">\n        Jobs\n      </th>\n      <th>\n        Printer(s)\n      </th>\n      <th>\n        Amount of prints\n      </th>\n      <th>\n        Created at\n      </th>\n    </tr>\n  </thead... (via streamed from JobListChannel)
```

I would love to hear if this is ready to merge, or if it needs a little more work.

dhh commented:
This would give us three dots even when you're not truncating. You can use String#truncate from Active Support to do this.

jankeesvw commented:
I changed it, now it uses ActiveSupport's `truncate` method. Is this ready to merge?

dhh commented:
You don't actually need to specify the omission part. Three dots is the default. So it can just be truncate(300).

jankeesvw commented:
Whoops, sorry! Changed!

dhh commented:
👌
